urbanopt/geojson-modelica-translator
# Schema changes

#### Any background context you want to provide?
This addresses points 1 & 2 of [this comment](https://github.com/urbanopt/geojson-modelica-translator/issues/401#issuecomment-896323096) from #401 by organizing our sys-params slightly differently
#### What does this PR accomplish?
- Refactors templates to pull sys-param data that is only central-plant related from the district block of the sys-param file
- Adds district blocks to test sys-param files (it's possible I added district blocks in a few places where they're not needed. I don't think that harms anything, if there were some extra json lines)
- Removes outdated params from schema and test sys-param files
#### How should this be manually tested?
`poetry run py.test tests/model_connectors/test_district_heating_system.py` should return a different value for `T_a1_nominal` in the `Substation/HeatingIndirect_<id>.mo` file when you change the `temp_setpoint_hhw` parameter in `time_series_system_params_ets.json`
#### Other thoughts
There are district params that are still in the ETS section of sys-params. @amyeallen1 confirmed that the remaining `..._district` parameters are appropriate in the ETS block.
